frodo-runtime/src/main/java/com/fernandocejas/frodo/aspect/LogSubscriber.java

```diff
@@ -78,9 +80,14 @@ public void beforeOnStartExecution(JoinPoint joinPoint) {
   public void beforeOnNextExecution(JoinPoint joinPoint) {
     countAndMeasureTime();
     final FrodoJoinPoint frodoJoinPoint = new FrodoJoinPoint(joinPoint);
-    final Object value = frodoJoinPoint.getMethodParamValuesList().get(0);
-    messageManager.printSubscriberOnNext(joinPoint.getTarget().getClass().getSimpleName(),
-        value.toString(),
+    final String message;
```

Comment:
It is not this class responsibility to build the message. This logic should sit on ```MessageManager.class```. It might be a good idea to pass in the value as it is: ```Object.class``` and handle the ```null``` case inside ```printSubscriberOnNext()``` method.